projects/1.19/assets/heat-and-climate/dcs_climate/lang/zh_cn.json

```
@@ -868,6 +904,15 @@
 
   "dcs.tip.null_item1":"没有物品被注册到这个配方",
   "dcs.tip.null_item2":"请检查此配方的配置文件",
+  
+  "dcs.tip.container.ownable_short":"锁定（按下左shift展开提示）",
+  "dcs.tip.container.ownable":"所有者可以锁定此方块，使它只能被所有者和服务器管理使用，同时阻止漏斗的输入输出",
+  "dcs.tip.container.ownable_locked":"方块已被 %1$s 锁定",

```

```suggestion
  "dcs.tip.container.ownable_locked":"方块已被%1$s锁定",
```
据《翻译规范》，不必添加空格
建议实际进游戏看一下效果，或者问一下作者原文加空格的用途